static/monitoring.js

```diff
@@ -245,24 +289,13 @@ function fetchDevices() {
             info.appendChild(create("h3", "Attestation history"));
             appendLine(info, "First verified time: " + new Date(device.verifiedTimeFirst));
             appendLine(info, "Last verified time: " + new Date(device.verifiedTimeLast));
+
             info.appendChild(create("button", "show detailed history", "toggle"));
+
             const history = info.appendChild(document.createElement("div"));
+            history.deviceFingerprint = device.fingerprint;
             history.className = "hidden";
-
-            for (const attestation of device.attestations) {
```

Comment:
Since this isn't being used anymore, it should be removed from the devices.json API. We don't provide any kind of API stability guarantees and it's very bloated data to be sending over the network especially when it's usually not being viewed.